app/destination/components/NavItem/page.js

```diff
@@ -0,0 +1,12 @@
+import Link from "next/link";
+
+export const NavItem = ({ title, link, isActive, index }) => {
+  return (
+    <>
+      <Link href={link}>
+        <b>{`0${index}`}</b> {title}
+      </Link>
+      {{ link } ? isActive : "/"}
```

Comment:
This just adds in some extra characters to the DOM, I think you can delete it.

Reply:
It was the extra component I have created for using it in NavBarItem with map()